baystation12.dme

```diff
@@ -3629,8 +3630,13 @@
 #include "code\unit_tests\~helpers.dm"
 #include "interface\interface.dm"
 #include "interface\skin.dmf"
+#include "maps\_map_include.dm"
 #include "maps\_maps.dm"
 #include "maps\away\icarus\icarus_areas.dm"
+#include "maps\away\geminus_city\road_and_pavements.dm"
+#include "maps\away\geminus_city\signs.dm"
+#include "maps\away\geminus_city\streetobjects.dm"
+#include "maps\away\glloyd_jungle\glloyd_jungle.dm"

```

Comment:
why was this moved to the .dme? It might cause issues loading glloydstation.

Reply:
You told me to in the discord :P 
It was moved because it was required in `away_site_testing`, but this was only ever included in the Nerva code. If it's in the .dme, all maps will have access to it, so it shouldn't cause issues for glloydstation?

Reply:
i'm referring to glloyd_jungle.dm. The geminus stuff is fine to be in the dme, but that shouldn't because it's the actual awaymap definition. The jungle on glloydstation is just part of the map, not an away.

Reply:
On that thread though, there are a couple nerva awaymaps missing from away_site_testing it seems

Reply:
Ah. You know, I don't remember why. I'll re-run some tests with it back and get back to you

Reply:
Re: Missing awaymaps, as said on a different PR, I'm tempted to write a mini test that'll scream if an awaymap isn't included in `away_sites_testing.dm`. I assume basically all aways should be in there?

Reply:
yeah, i can't think of any reason why one shouldn't be in there as long as it's in use.

Reply:
No idea why I moved the jungle. My bad. Moved it back to Nerva & added it to away testing